src/WingetCreateCLI/Commands/ShowCommand.cs

```diff
@@ -0,0 +1,212 @@
+            }
+        }
+
+        private static void ShowAllManifests(Manifests manifests)
+        {
+            DisplayInstallerManifest(manifests.InstallerManifest);
+            DisplayDefaultLocaleManifest(manifests.DefaultLocaleManifest);
+            DisplayLocaleManifests(manifests.LocaleManifests);
+            DisplayVersionManifest(manifests.VersionManifest);
+        }
+
+        private static void DisplayInstallerManifest(InstallerManifest installerManifest)
+        {
+            Logger.InfoLocalized(nameof(Resources.InstallerManifest_Message));
+            Console.WriteLine(installerManifest.ToYaml(true));
+        }
+
+        private static void DisplayVersionManifest(VersionManifest versionManifest)
+        {
+            Logger.InfoLocalized(nameof(Resources.VersionManifest_Message));
+            Console.WriteLine(versionManifest.ToYaml(true));
+        }
+
+        private static void DisplayDefaultLocaleManifest(DefaultLocaleManifest defaultLocaleManifest)
+        {
+            Logger.InfoLocalized(nameof(Resources.DefaultLocaleManifest_Message), defaultLocaleManifest.PackageLocale);
+            Console.WriteLine(defaultLocaleManifest.ToYaml(true));
+        }
+
+        private static void DisplayLocaleManifests(List<LocaleManifest> localeManifests)
+        {
+            foreach (var localeManifest in localeManifests)
+            {
+                Logger.InfoLocalized(nameof(Resources.LocaleManifest_Message), localeManifest.PackageLocale);
+                Console.WriteLine(localeManifest.ToYaml(true));
+            }
+        }
+
+        private static void DisplaySingletonManifest(SingletonManifest singletonManifest)
+        {
+            Logger.InfoLocalized(nameof(Resources.SingletonManifest_Message));
+            Console.WriteLine(singletonManifest.ToYaml(true));
+        }
+
+        private void ParseArgumentsAndShowManifest(Manifests manifests)
+        {
+            if (manifests.SingletonManifest != null)
+            {
+                DisplaySingletonManifest(manifests.SingletonManifest);
+                return;
+            }
+
+            bool showAll = !this.ShowInstallerManifest && !this.ShowDefaultLocaleManifest && !this.ShowLocaleManifests && !this.ShowVersionManifest;
+            if (this.ShowInstallerManifest)
+            {
+                DisplayInstallerManifest(manifests.InstallerManifest);
+            }
+
+            if (this.ShowDefaultLocaleManifest || this.ShowLocaleManifests)
+            {
+                DisplayDefaultLocaleManifest(manifests.DefaultLocaleManifest);
+            }
+
+            if (this.ShowLocaleManifests)
+            {
+                DisplayLocaleManifests(manifests.LocaleManifests);
+            }
+
+            if (this.ShowVersionManifest)
+            {
+                DisplayVersionManifest(manifests.VersionManifest);
+            }
+
+            if (showAll)
+            {
+                ShowAllManifests(manifests);
+            }
```

Comment:
I would move this up to the beginning and return after showing all the manifests since the rest of the args don't need to be checked.

Reply:
Check moved above